site/src/components/pages/hub/block-data-container.tsx

```diff
@@ -40,6 +40,11 @@ type BlockDataContainerProps = {
 
 const validator = new Validator();
 
+/** Blocks that aren't compliant with BP V0.2  */
+export const checkIsUnsupportedBlock = ({ protocol }: BlockMetadata) => {
+  return !(parseFloat(protocol) >= 0.2);
```

Comment:
We are missing a check for 0.3+ (which would be invalid for the hub). This is only a theoretical issue though, so we can potentially ignore it for now. I'd go for something like:

```
const checkIfBlockIsSupported = ({ protocol }: BlockMetadata) => protocol !== "0.2"
````
	
We don't need to export this fn. If we want to share it, it'd have to go into some other file. This one should only export  the component and its props.